examples/json_logger.rs

```diff
@@ -0,0 +1,34 @@
+#[macro_use]
+extern crate log;
+
+use log::LevelFilter;
+use log4rs::append::console::ConsoleAppender;
+use log4rs::config::{Appender, Root};
+use log4rs::encode::json::JsonEncoder;
+
+//refresh_rate: 30 seconds
```

Comment:
This will make the config different than the runtime built one

Reply:
Oh true. I'll remove this to keep parity with the programmatic one.